management_app/templates/courses/catalog.html

```diff
@@ -42,7 +42,11 @@
     {% for course in courses %}
       <tr>
         <th scope="row">{{ course['course_title_id'] }}</th>
-        <td>{{ course['course_title'] }}</td>
+        {% if course['combine_with'] is not none %}
+          <td>{{ course['course_title'] }} (Combined with {{ course['combine_with'] }})</td>
```

Comment:
I also did this in my latest pull request. Let me know which do you prefer.

Reply:
Initially I was thinking about adding a combine with column too but decided against it because it doesn't come up often so the column will be blank for entries most of the time. That's why I just put it in the title but I think your way works too. Let's go with your way. I'll undo my commit for this now.